From 411d9c466b356986ab53e7eb6ad806dd1586b569 Mon Sep 17 00:00:00 2001
Message-Id: <411d9c466b356986ab53e7eb6ad806dd1586b569.1425610844.git.chang-joon.lee@intel.com>
In-Reply-To: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
References: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Sat, 14 Feb 2015 15:55:23 +0530
Subject: [PATCH 15/27] SQUASHME! [VPG]: drm/i915/adf: Respect adf kernel
 parameter.

Do adf wrapper init only if adf driver is enabled as kernel parameter.

Squash to below commit:
Author: Deepak S <deepak.s@intel.com>
Date:   Sun Aug 24 10:13:46 2014 +0530
Change-Id: I85067e2523217e300ad3ef7a2e4d3c14d32378f5
    MUST_REBASE [VPG]: drm/i915/adf: Added a wrapper between i915 and
    adf.

Issue: GMINL-3547
Change-Id: I1e32a087875efa29539aa615898b37b094cbd5d4
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_adf_wrapper.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/gpu/drm/i915/i915_adf_wrapper.c b/drivers/gpu/drm/i915/i915_adf_wrapper.c
index 1df72ec..fec5b39 100644
--- a/drivers/gpu/drm/i915/i915_adf_wrapper.c
+++ b/drivers/gpu/drm/i915/i915_adf_wrapper.c
@@ -52,6 +52,9 @@ static struct drm_i915_private *i915_adf_dev;
 
 void i915_adf_wrapper_init(struct drm_i915_private *dev_priv)
 {
+	if (!i915.enable_intel_adf)
+		return;
+
 	i915_adf_dev = dev_priv;
 
 	/*
-- 
1.7.9.5

